abci/proposal_auction.go

```diff
@@ -88,12 +88,13 @@ func (h *ProposalHandler) VerifyTOB(ctx sdk.Context, proposalTxs [][]byte) (*Auc
 
 	// verify that the included vote extensions are valid in accordance with the
 	// the preferences of the application
-	if err := h.validateVoteExtensionsFn(ctx, ctx.BlockHeight(), lastCommitInfo); err != nil {
+	cacheCtx, _ := ctx.CacheContext()
```

Comment:
There is a simple two step process here that will explain why I use a cache ctx. 1. by using a cache context, we can simplify alot of the TOB custom code by being able to inherit the logic of the tob lane instead. 2. cache context keeps context clean so that we can reuse it out of the box in process proposal.

Reply:
👍